### ISSUE TITLE
Bulk method should have an accessibility method for building bulk body.

### ISSUE THREAD

**Issue by siddharthlatest** [appbaseio/appbase-droid#6]
Reference on how Elasticsearch does it - https://www.elastic.co/guide/en/elasticsearch/client/java-api/2.4/java-docs-bulk.html.
